stdpopsim/catalog/DroSec/genome_data.py

```diff
@@ -1,13 +1,6 @@
-# File created manually from https://www.ncbi.nlm.nih.gov/assembly/GCF_004382195.1
+# File autogenerated from Ensembl REST API. Do not edit.
```

Comment:
Looks like this file shouldn't be changed as was created manually

Reply:
maybe we should add a species level attribute -- `manually_added` or something -- that would indicate if the maintenance script should go ahead with the download

Reply:
I'd vote "no", so we are alerted when a species that previously didn't have a build gets one; otherwise just manually back out the change in such cases.

(btw the reason this doesn't work is because [ensembl doesn't have a karyotype entry](http://rest.ensembl.org/info/assembly/drosophila_sechellia?synonyms=1) for DroSec, just some ~Mb scale scaffolds)